src/linking/linkModule.ml

```diff
@@ -846,7 +846,13 @@ let link (em1 : extended_module) libname (em2 : extended_module) =
     )
   end else ();
 
-  let old_table_size = read_table_size em1.module_ in
+  let max x y = if x >= y then x else y in
```

Comment:
```suggestion
  let max x y = if x >= y then x else y in (* use `Int.max` when bumping to 4.13 *)
```

Reply:
Thank you. I also noticed it would be in Ocaml 4.13. Good to have a todo comment in the code.